charts/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -64,9 +64,12 @@ spec:
         {{- end }}
         {{- end }}
     spec:
-      {{- with .Values.imagePullSecrets }}
+      {{- if .Values.master.imagePullSecrets}}
       imagePullSecrets:
-        {{- toYaml . | nindent 8 }}
+        {{- toYaml .Values.master.imagePullSecrets | nindent 8 }}
+      {{- else if .Values.imagePullSecrets}}
```

Comment:
Add a whitespace before }} in the template directive.

Reply:
use `fuse` not `master` imagePullSecrets

Reply:
> use `fuse` not `master` imagePullSecrets

thanks for point that, have amended

Reply:
> Add a whitespace before }} in the template directive.

ok, have done